scripts/http/public.http

```diff
@@ -6,6 +6,13 @@
 @x-rh-identity-system = ewogICJpZGVudGl0eSI6IHsKICAgICJhY2NvdW50X251bWJlciI6ICIxMTExMSIsCiAgICAiYXV0aF90eXBlIjogImNlcnQtYXV0aCIsCiAgICAiaW50ZXJuYWwiOiB7CiAgICAgICJhdXRoX3RpbWUiOiA5MDAsCiAgICAgICJjcm9zc19hY2Nlc3MiOiBmYWxzZSwKICAgICAgIm9yZ19pZCI6ICIxMjM0NSIKICAgIH0sCiAgICAib3JnX2lkIjogIjEyMzQ1IiwKICAgICJzeXN0ZW0iOiB7CiAgICAgICJjZXJ0X3R5cGUiOiAic3lzdGVtIiwKICAgICAgImNuIjogIjZmMzI0MTE2LWIzZDItMTFlZC04YTM3LTQ4MmFlMzg2M2QzMCIKICAgIH0sCiAgICAidHlwZSI6ICJTeXN0ZW0iCiAgfSwKICAiZW50aXRsZW1lbnRzIjogewogICAgImFuc2libGUiOiB7CiAgICAgICJpc19lbnRpdGxlZCI6IHRydWUsCiAgICAgICJpc190cmlhbCI6IGZhbHNlCiAgICB9LAogICAgImNvc3RfbWFuYWdlbWVudCI6IHsKICAgICAgImlzX2VudGl0bGVkIjogdHJ1ZSwKICAgICAgImlzX3RyaWFsIjogZmFsc2UKICAgIH0sCiAgICAiaW5zaWdodHMiOiB7CiAgICAgICJpc19lbnRpdGxlZCI6IHRydWUsCiAgICAgICJpc190cmlhbCI6IGZhbHNlCiAgICB9LAogICAgImludGVybmFsIjogewogICAgICAiaXNfZW50aXRsZWQiOiBmYWxzZSwKICAgICAgImlzX3RyaWFsIjogZmFsc2UKICAgIH0sCiAgICAibWlncmF0aW9ucyI6IHsKICAgICAgImlzX2VudGl0bGVkIjogdHJ1ZSwKICAgICAgImlzX3RyaWFsIjogZmFsc2UKICAgIH0sCiAgICAib3BlbnNoaWZ0IjogewogICAgICAiaXNfZW50aXRsZWQiOiB0cnVlLAogICAgICAiaXNfdHJpYWwiOiBmYWxzZQogICAgfSwKICAgICJyaGVsIjogewogICAgICAiaXNfZW50aXRsZWQiOiB0cnVlLAogICAgICAiaXNfdHJpYWwiOiBmYWxzZQogICAgfSwKICAgICJyaG9hbSI6IHsKICAgICAgImlzX2VudGl0bGVkIjogZmFsc2UsCiAgICAgICJpc190cmlhbCI6IGZhbHNlCiAgICB9LAogICAgInJob2RzIjogewogICAgICAiaXNfZW50aXRsZWQiOiBmYWxzZSwKICAgICAgImlzX3RyaWFsIjogZmFsc2UKICAgIH0sCiAgICAicmhvc2FrIjogewogICAgICAiaXNfZW50aXRsZWQiOiBmYWxzZSwKICAgICAgImlzX3RyaWFsIjogZmFsc2UKICAgIH0sCiAgICAic2V0dGluZ3MiOiB7CiAgICAgICJpc19lbnRpdGxlZCI6IHRydWUsCiAgICAgICJpc190cmlhbCI6IGZhbHNlCiAgICB9LAogICAgInNtYXJ0X21hbmFnZW1lbnQiOiB7CiAgICAgICJpc19lbnRpdGxlZCI6IHRydWUsCiAgICAgICJpc190cmlhbCI6IGZhbHNlCiAgICB9LAogICAgInN1YnNjcmlwdGlvbnMiOiB7CiAgICAgICJpc19lbnRpdGxlZCI6IHRydWUsCiAgICAgICJpc190cmlhbCI6IGZhbHNlCiAgICB9LAogICAgInVzZXJfcHJlZmVyZW5jZXMiOiB7CiAgICAgICJpc19lbnRpdGxlZCI6IHRydWUsCiAgICAgICJpc190cmlhbCI6IGZhbHNlCiAgICB9CiAgfQp9Cg==
+# Generated by:
+# ./tools/bin/xrhidgen \
+#   -org-id 12345 -account-number 11111 -employe-account-number 22222 \
+#   service-account \
+#   -username jdoe -client-id "91a229a3-ea92-4ec9-a926-8885d67e6a56"
+@x-rh-identity-service-account = eyJpZGVudGl0eSI6eyJhY2NvdW50X251bWJlciI6IjExMTExIiwiZW1wbG95ZWVfYWNjb3VudF9udW1iZXIiOiIyMjIyMiIsIm9yZ19pZCI6IjEyMzQ1IiwiaW50ZXJuYWwiOnsib3JnX2lkIjoiMTIzNDUifSwidXNlciI6eyJ1c2VybmFtZSI6Impkb2UiLCJlbWFpbCI6Impkb2VAZXhhbXBsZS5jb20iLCJmaXJzdF9uYW1lIjoiSm9obiIsImxhc3RfbmFtZSI6IkRvZSIsImlzX2FjdGl2ZSI6dHJ1ZSwiaXNfb3JnX2FkbWluIjp0cnVlLCJpc19pbnRlcm5hbCI6dHJ1ZSwibG9jYWxlIjoiZW4iLCJ1c2VyX2lkIjoiMTk4NzM0OCJ9LCJzeXN0ZW0iOnt9LCJhc3NvY2lhdGUiOnsiUm9sZSI6bnVsbCwiZW1haWwiOiIiLCJnaXZlbk5hbWUiOiIiLCJyaGF0VVVJRCI6IiIsInN1cm5hbWUiOiIifSwieDUwOSI6eyJzdWJqZWN0X2RuIjoiIiwiaXNzdWVyX2RuIjoiIn0sInR5cGUiOiJVc2VyIiwiYXV0aF90eXBlIjoiIn19Cg==
```

Comment:
This identity is not service account. It's a copy&paste of the user account. Same issue in `rbac.http`.

```
 echo "eyJpZGVudGl0eSI6eyJhY2NvdW50X251bWJlciI6IjExMTExIiwiZW1wbG95ZWVfYWNjb3VudF9udW1iZXIiOiIyMjIyMiIsIm9yZ19pZCI6IjEyMzQ1IiwiaW50ZXJuYWwiOnsib3JnX2lkIjoiMTIzNDUifSwidXNlciI6eyJ1c2VybmFtZSI6Impkb2UiLCJlbWFpbCI6Impkb2VAZXhhbXBsZS5jb20iLCJmaXJzdF9uYW1lIjoiSm9obiIsImxhc3RfbmFtZSI6IkRvZSIsImlzX2FjdGl2ZSI6dHJ1ZSwiaXNfb3JnX2FkbWluIjp0cnVlLCJpc19pbnRlcm5hbCI6dHJ1ZSwibG9jYWxlIjoiZW4iLCJ1c2VyX2lkIjoiMTk4NzM0OCJ9LCJzeXN0ZW0iOnt9LCJhc3NvY2lhdGUiOnsiUm9sZSI6bnVsbCwiZW1haWwiOiIiLCJnaXZlbk5hbWUiOiIiLCJyaGF0VVVJRCI6IiIsInN1cm5hbWUiOiIifSwieDUwOSI6eyJzdWJqZWN0X2RuIjoiIiwiaXNzdWVyX2RuIjoiIn0sInR5cGUiOiJVc2VyIiwiYXV0aF90eXBlIjoiIn19Cg==" | base64 -d | jq
{
  "identity": {
    "account_number": "11111",
    "employee_account_number": "22222",
    "org_id": "12345",
    "internal": {
      "org_id": "12345"
    },
    "user": {
      "username": "jdoe",
      "email": "jdoe@example.com",
      "first_name": "John",
      "last_name": "Doe",
      "is_active": true,
      "is_org_admin": true,
      "is_internal": true,
      "locale": "en",
      "user_id": "1987348"
    },
    "system": {},
    "associate": {
      "Role": null,
      "email": "",
      "givenName": "",
      "rhatUUID": "",
      "surname": ""
    },
    "x509": {
      "subject_dn": "",
      "issuer_dn": ""
    },
    "type": "User",
    "auth_type": ""
  }
}
```

Reply:
:+1: good catch!!!  thanks for this!

Reply:
The issue seems to be still present in the `rbac.http` (as also mentioned in prev. comment).

Reply:
thanks for the heads up! I am on it!